src/components/Shows/useShows.ts

```diff
@@ -1,12 +1,17 @@
 import { useQuery } from 'react-query';
 import { Show } from './types';
+import config from 'react-native-config';
 
 export const useShows = () => {
   const { data, isLoading, error } = useQuery(
     'getShows',
     async () => {
       try {
-        const resp = await fetch('https://arcsi.lahmacun.hu/arcsi/show/all');
+        const resp = await fetch('https://arcsi.lahmacun.hu/arcsi/show/all', {
```

Comment:
We have more efficient APIs than this one, maybe we can start to use them later on.

Reply:
Yep, that's true! The app passes down show and episode info from this request, so changing it would require some refactor. I'll make a ticket.
Thanks!